modules/notify/spec/lib/service_spec.rb

```diff
@@ -0,0 +1,129 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'notify/service'
+
+describe Notify::Service do
+  before(:example, test_service: false) do
+    @test_api_key = 'test-aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa-bbbbbbbb-bbbb-bbbb-bbbb-bbbbbbbbbbbb'
+    @test_base_url = 'http://fakeapi.com'
+    allow_any_instance_of(Notify::Configuration).to receive(:base_path).and_return(@test_base_url)
+  end
+
+  send_email_parameters = {
```

Comment:
I'm wondering if you should wrap the hash into a before(:each)? Prob nitpicky ha

Reply:
Yeah, I was just copying over what already existed, but that is probably a good call.